pkg/app/app.go

```diff
@@ -17,6 +17,25 @@ import (
 	"github.com/snyk/go-application-framework/pkg/workflow"
 )
 
+func defaultFunc_FF_CODE_CONSISTENT_IGNORES(engine workflow.Engine, config configuration.Configuration, apiClient api.ApiClient, logger *zerolog.Logger) configuration.DefaultValueFunction {
+	callback := func(existingValue interface{}) interface{} {
```

Comment:
question: are we ok not defining an interface and using `interface{}` here? It feels analogous to TypeScript's `any` which is a bit of a bad habit one can fall into

Reply:
We are just aligning with [Vipers Get() interface](https://github.com/spf13/viper/blob/master/viper.go#L900C22-L900C25)